.github/workflows/deploy-staging.yml

```diff
@@ -1,67 +0,0 @@
-name: Deploy staging
```

Comment:
Sweet! I was going to complain about "Deploy staging" being broken on all my pushes :)